common/cache/namespaceCache.go

```diff
@@ -65,6 +66,8 @@ const (
 	namespaceCacheInitialSize = 10 * 1024
 	namespaceCacheMaxSize     = 64 * 1024
 	namespaceCacheTTL         = 0 // 0 means infinity
+	// NamespaceCacheMinRefreshInterval namespace cache refresh interval
```

Comment:
"Minimun namespace cache refresh interval" rather?

Reply:
the naming convention (at least) in this file is `namespaceCache` + var

Reply:
I think @sergeybykov wanted this:
```suggestion
	// NamespaceCacheMinRefreshInterval is a minimun namespace cache refresh interval.
```

Reply:
my bad